sections/collapsible-content.liquid

```diff
@@ -0,0 +1,382 @@
+{{ 'component-accordion.css' | asset_url | stylesheet_tag }}
+{{ 'collapsible-content.css' | asset_url | stylesheet_tag }}
+
+<div class="collapsible-content{% if section.settings.layout == 'no_container' %} color color-{{ section.settings.color_scheme }}{% endif %}{% unless section.settings.layout == 'no_container' %} color-{{ section.settings.container_color_scheme }}{% endunless %}{% if section.settings.layout == 'section_container' %} page-width{% endif %}">
+  <div class="collapsible-content-wrapper{% if section.settings.layout == 'section_container' %} color color-{{ section.settings.color_scheme }}{% endif %}">
+    <div class="{% if section.settings.image == blank %} collapsible-content-wrapper--narrow{% else %} page-width{% endif%}">
+      <div style="text-align: {{ section.settings.heading_alignment }}">
+        {%- if section.settings.caption != blank -%}
+          <p class="caption-with-letter-spacing"> {{ section.settings.caption }}</p>
+        {% endif %}
+        {%- if section.settings.heading != blank -%}
+          <h2 class="h1">{{ section.settings.heading }}</h2>
```

Comment:
Are we able to [add a condition to implement a fallback heading `h2`](https://github.com/Shopify/dawn/pull/956#discussion_r760551926) for when title text is not provided?